### PR TITLE
chore: cleanup and update unit tests

### PR DESCRIPTION
**Pull request by amsmith-pro**

### CODE REVIEW

File: client-grpc/src/grpc.rs [hunk 1]
```diff
@@ -401,6 +401,7 @@ impl ZoneType {
     }
 }
 /// Generated client implementations.
+#[cfg(not(tarpaulin_include))]
```
Comment (owlot) [inline client-grpc/src/grpc.rs:404]: missing reason

File: server/src/cache/mod.rs [hunk 2]
```diff
@@ -57,10 +55,12 @@ where
     fn sleep_ms(&self) -> u64;
 
     /// Starts a loop to consume data from the Redis queue
+    #[cfg(not(tarpaulin_include))]
+    // no_coverage: loop, needs redis backend to test
```
Comment (owlot) [inline server/src/cache/mod.rs:59]: Your script to test the no_coverage reasons requires a release tag. I've used the tag `Rnever` for code that can never be tested

File: server/src/postgis/mod.rs [hunk 3]
```diff
@@ -102,39 +99,34 @@ impl std::fmt::Display for PsqlError {
...
-    fn source(&self) -> Option<&(dyn std::error::Error + 'static)> {
-        None
-    }
-}
+impl std::error::Error for PsqlError {}
 
 /// Executes a transaction with multiple statements on the provided pool
 ///  with rollback if any of the statements fail to execute.
+#[cfg(not(tarpaulin_include))]
+// no_coverage: need running postgresql instance
```
Comment (owlot) [inline server/src/postgis/mod.rs:107]: Missing release tag

File: server/src/postgis/mod.rs [hunk 4]
```diff
@@ -168,12 +160,15 @@ where
...
-    postgis_info!("(psql_enum_declaration) {}.", declaration);
+
+    postgis_info!("{}.", declaration);
 
     declaration
 }
 
 /// Initializes the PostgreSQL database with the required tables and enums
+#[cfg(not(tarpaulin_include))]
+// no_coverage: need running postgresql instance
```
Comment (owlot) [inline server/src/postgis/mod.rs:171]: Missing release tag

File: server/src/postgis/nearest.rs [hunk 5]
```diff
@@ -126,14 +127,15 @@ async fn nearest_neighbor_aircraft_source(
 
 /// Nearest neighbor query for nodes
 #[cfg(not(tarpaulin_include))]
+// no_coverage: need to run on a gis database
```
Comment (owlot) [inline server/src/postgis/nearest.rs:130]: Missing release tag

Comment (amsmith-pro) [reply server/src/postgis/nearest.rs]: Fixed, see new release checklist generated

File: server/src/postgis/mod.rs (hunk 4 above)
Comment (amsmith-pro) [reply server/src/postgis/mod.rs]: Fixed, see new release checklist generated

File: server/src/postgis/mod.rs (hunk 3 above)
Comment (amsmith-pro) [reply server/src/postgis/mod.rs]: Fixed, see new release checklist generated

File: server/src/cache/mod.rs (hunk 2 above)
Comment (amsmith-pro) [reply server/src/cache/mod.rs]: Fixed, see new release checklist generated

File: client-grpc/src/grpc.rs (hunk 1 above)
Comment (amsmith-pro) [reply client-grpc/src/grpc.rs:404]: Fixed, see new release checklist generated

**Review by owlot — APPROVED** [review]

**Pull request merged**